src/statsmodel.jl

```diff
@@ -195,16 +195,15 @@ end
 
 # show function that delegates to coeftable
 function Base.show(io::IO, model::TableModels)
+    println(io, typeof(model))
+    println(io)
+    println(io, model.mf.f)
+    println(io)
     try
-        ct = coeftable(model)
-        println(io, typeof(model))
-        println(io)
-        println(io, model.mf.f)
-        println(io)
         println(io,"Coefficients:")
-        show(io, ct)
+        show(io, coeftable(model))
     catch e
-        if isa(e, ErrorException) && occursin("coeftable is not defined", e.msg)
+        if isa(e, MethodError) || isa(e, ErrorException) && occursin("coeftable is not defined", e.msg)
             show(io, model.model)
```

Comment:
In addition to checking for a `MethodError` (so that we support the last several StatsBase/StatsAPI versions), I re-arranged things to that only the `coeftable` was in the try/catch block. Then we skip the re-throw for errors happening in the other statements.